index.bs

```diff
@@ -204,14 +204,18 @@ NOTE: The dimensions indicated in the 'ispe' property might not match the values
 
 NOTE: The values of [=render_width_minus1=] and [=render_height_minus1=] possibly present in the AV1 bistream are not exposed in the AVIF container level.
 
+<h4 id="clean-aperture-property">Clean Aperture Property</h4>
+
+The semantics of the clean aperture property ('clap') as defined in [[!HEIF]] apply. In addition to the restrictions on transformative item property ordering specified in [[!MIAF]], the following restriction also applies:
+
+<assert>The origin of the 'clap' item property shall be anchored to 0,0 (top-left) unless the full, un-cropped image item is included as a secondary item in an image collection.</assert>
```

Comment:
It would also be convenient to explicitly say that a reader may ignore any 'clap' in all cases because the full image is always included in the same `altr` group (if I understood this PR correctly).

Reply:
No, I don't think that's the same thing.

Say we have item 1 that is the base item.
We also have items 2, 3 and 4 which are `iden` items pointing to 1 that adds a `clap` to crop out specific people in the image.
As a last step, we create item 5, an `iovl` that composits items 2-4 together into some kind of collage.

I don't necessarily want to make items 1-4 alternatives of each other. And I don't want item 5 to ignore the `clap` on items 2-4.